boards/xtensa/esp32_ethernet_kit/doc/index.rst

```diff
@@ -0,0 +1,6 @@
+.. _esp32_ethernet_kit:
+
+ESP32-ETHERNET-KIT
+##############
+
+https://docs.espressif.com/projects/esp-idf/en/latest/esp32/hw-reference/esp32/get-started-ethernet-kit.html
```

Comment:
Documentation for this new board is currently lacking.
Is the procedure to copy/paste/edit from a similar board?

Reply:
Yes, you canc opy most of the setup part of the default ESP32. You can also use some specific sample code which uses ethernet as reference. If possible, add more information regarding the ethernet-devkit board itself, similar to https://docs.zephyrproject.org/3.1.0/boards/xtensa/esp_wrover_kit/doc/index.html